web-app/src/main/webapp/index.html

```diff
@@ -34,10 +34,163 @@
+        </div>
+        <div class="three columns desc">
+          months
+        </div>
+      </div>
+      <!-- Category -->
+      <div class="row menu-subheading">
+        <h2>Category</h2>
+      </div>
+      <div class="row">
+        <div class="two columns right">
+          <input type="checkbox" id="p-one" name="category"
+            value="p-one" checked="checked">
+        </div>
+        <div class="ten columns">
+          <label for="p-one">Placeholder 1</label>
+        </div>
+      </div>
+      <div class="row">
+        <div class="two columns right">
+          <input type="checkbox" id="p-two" name="category"
+            value="p-two" checked="checked">
+        </div>
+        <div class="ten columns">
+          <label for="p-two">Placeholder 2</label>
+        </div>
+      </div>
+      <div class="row">
+        <div class="two columns right">
+          <input type="checkbox" id="p-three" name="category"
+            value="p-three" checked="checked">
+        </div>
+        <div class="ten columns">
+          <label for="p-three">Placeholder 3</label>
+        </div>
+      </div>
+      <div class="row">
+        <div class="two columns right">
+          <input type="checkbox" id="p-four" name="category"
+            value="p-four" checked="checked">
+        </div>
+        <div class="ten columns">
+          <label for="p-four">Placeholder 4</label>
+        </div>
+      </div>
+      <button id="select-all-cats" class="menu-button">Select all</button>
+      <!-- Time of day -->
+      <div class="row menu-subheading">
+        <h2>Time of Day</h2>
+      </div>
+      <div class="row">
+        <div class="two columns right">
+          <input type="checkbox" id="morning" name="time-of-day"
+            value="morning" checked="checked">
+        </div>
+        <div class="ten columns">
+          <label for="morning">Morning</label>
+        </div>
+      </div>
+      <div class="row">
+        <div class="two columns right">
+          <input type="checkbox" id="afternoon" name="time-of-day"
+            value="afternoon" checked="checked">
+        </div>
+        <div class="ten columns">
+          <label for="afternoon">Afternoon</label>
+        </div>
+      </div>
+      <div class="row">
+        <div class="two columns right">
+          <input type="checkbox" id="evening" name="time-of-day"
+            value="evening" checked="checked">
+        </div>
+        <div class="ten columns">
+          <label for="evening">Evening</label>
+        </div>
+      </div>
+      <div class="row">
+        <div class="two columns right">
+          <input type="checkbox" id="night" name="time-of-day"
+            value="night" checked="checked">
+        </div>
+        <div class="ten columns">
+          <label for="night">Night</label>
+        </div>
+      </div>
+      <button id="select-all-times" class="menu-button">Select all</button>
```

Comment:
Were we gonna let them choose time of the day still?? (Since police reports only have until months for time)

Reply:
I think we can still use time of day for user generated reports, maybe leave it in for now?